### PR TITLE
#7968 create tutorial index.md

### LINKED ISSUE #7968: Suggest a tutorial
### Tutorial title

Connect your website to MetaMask and ENS

### Tutorial description

Learn how to fetch the user's account data and possible ENS domain using JavaScript and the ethers.js library and thus making it accessible to your HTML website.

### Tutorial tags

metamask, ens, ethersjs, ethers.js, javascript, html, css

### Skill level

Intermediate

### Hosted on ethereum.org or hosted elsewhere?

Hosted on ethereum.org

### For tutorials to be hosted on Ethereum.org: Tutorial Content

The tutorial contains descriptions of code snippets that will be displayed afterward for easy copy/paste access.
It is divided into 4 separate sections: HTML, CSS, JavaScript and Conclusion.
It's a simple tutorial to quickly access MetaMask through a website and extract the user's account data and ENS domain if there is one.
I have searched through the current tutorials and couldn't find a single one that offers this kind of knowledge.
That is why I strongly believe that my tutorial is a worthy addition.

### For tutorials hosted elsewhere: URL to tutorial

_No response_

### PR DESCRIPTION
**Pull request by SebastianSupreme**
## Description

I've created a simple tutorial on how to connect an HTML website to MetaMask using JavaScript and ethers.js to fetch the user's account data and possibly their ENS domain.

## Related Issue

#7968

### CODE REVIEW

File: src/content/developers/tutorials/implement-metamask-and-ens-into-website/index.md [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+published: 2022-09-23
+lang: en
+---
+
+In this article you will learn to easily implement MetaMask and Ethereum Name Service support into your website.
+This is accomplished using JavaScript and the ethers.js library.
+Please note that this tutorial is only for educational purposes and is not intended to be used commercially, but rather as a quick way to set up a makeshift implementation to test your code.
+The following code is licensed under the MIT license (SPDX-License-Identifier: MIT).
+
+Copyright (c) 2022 Sebastian Supreme
```
Comment (jmcook1186) [inline src/content/developers/tutorials/implement-metamask-and-ens-into-website/index.md:17]: all content on ethereum.org is available under a creative commons license (CC04).
```suggestion
```

File: src/content/developers/tutorials/implement-metamask-and-ens-into-website/index.md [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+sidebar: true
+tags: ["metamask", "ens", "ethersjs", "ethers.js", "javascript", "html", "css"]
+skill: intermediate
+published: 2022-09-23
+lang: en
+---
+
+In this article you will learn to easily implement MetaMask and Ethereum Name Service support into your website.
+This is accomplished using JavaScript and the ethers.js library.
+Please note that this tutorial is only for educational purposes and is not intended to be used commercially, but rather as a quick way to set up a makeshift implementation to test your code.
```
Comment (jmcook1186) [inline src/content/developers/tutorials/implement-metamask-and-ens-into-website/index.md:14]: disclaimers and licensing is all dealt with here https://ethereum.org/en/terms-of-use/#terms-of-use
```suggestion
```

**Comment by SebastianSupreme** [comment]
@jmcook1186
The changes you requested have been implemented👍

**Comment by SebastianSupreme** [comment]
@samajammin @wackerow @corwintines @pettinarip @minimalsm @jmcook1186 
Could you please take a quick look at this? I'm certain it would only take 5 minutes to review this and I would be more than happy if my PR finally merges. Thank you in advance and I'm sorry if I'm coming off as annoying, that's really not my intention.

**Comment by SebastianSupreme** [comment]
@samajammin @wackerow @corwintines @pettinarip @minimalsm @jmcook1186 
Could you please take a quick look at this? I'm certain it would only take 5 minutes to review and it would mean the world to me if my PR finally merges. Thank you in advance and I'm sorry if I'm coming off as annoying, that's really not my intention.

**Comment by samajammin** [comment]
Thanks @SebastianSupreme - I can assure you this PR is in the queue for us to review. I can also assure you that nagging in Discord & GitHub will not speed up the review process 😄

**Comment by corwintines** [comment]
Hey @SebastianSupreme.

Thanks for the PR, but I don't think this is a tutorial we need on ethereum.org.

I'm rejecting the tutorial for the following reasons:
1 - The code as it is wouldn't be used in a production application. If we were to have a tutorial on ENS resolving, I would like to see it include code that people would use in a production environment. This leads me to my next point.
2 - This tutorial only uses MetaMask, which is far too limiting in scope for something like this. If a tutorial for ENS is going to be added, it needs to be more general to the wallet landscape than this. I know there are lots of current tutorials that may have this same issue, but we are reassessing tutorials to improve this section of the site, and we are more likely to cut those than add more of them.
3 - ENS already has a great tutorial on this (https://docs.ens.domains/dapp-developer-guide/resolving-names). I would see more value linking to these docs as a tutorial than adding something that will be hard to maintain.

Thank you for the idea though!

**Comment by corwintines** [comment]
@all-contributors can you add @SebastianSupreme for idea

**Pull request closed without merging**